tools/accuracy_checker/configs/person-reidentification-retail-0300.yml

```diff
@@ -0,0 +1,43 @@
+models:
+  - name: person-reidentification-retail-0300
```

Comment:
@DmitrySidnev,  I right understand, is it replacemdent for person-reidentification-retail-0200?

Reply:
if yes, please also remove config for 0200

Reply:
removed

Reply:
removed